### PR TITLE
Issue 18068 remove references to load records to index

### PR DESCRIPTION
**Pull request by nollymar**
As the way we run the reindex process changed, the load_records_to_index procedure is not needed anymore, so it is being removed from code

Additionally, an issue on this task `Task05205UpdateIndexNameLength` was fixed. The upgrade process failed on MSSQL.

Besides, a couple of ITs were implemented to run upgrade tasks

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    private final String POSTGRES_SCRIPT = "DROP FUNCTION IF EXISTS load_records_to_index;";
+
+    private final String MYSQL_SCRIPT = "DROP PROCEDURE IF EXISTS load_records_to_index;";
+
+    private final String MSSQL_SCRIPT =
+            "IF EXISTS (SELECT * FROM sys.objects WHERE type = 'P' AND name = 'load_records_to_index')\n"
+                    + "DROP PROCEDURE load_records_to_index;\n"
+                    + "GO\n";
+
+    private final String ORACLE_SCRIPT = "DROP FUNCTION load_records_to_index";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+/**
+ * Remove {@code load_records_to_index} stored procedure
+ *
+ * @author nollymar
+ */
+public class Task05225RemoveLoadRecordsToIndex extends AbstractJDBCStartupTask {
+
+    private final String POSTGRES_SCRIPT = "DROP FUNCTION IF EXISTS load_records_to_index;";
+
+    private final String MYSQL_SCRIPT = "DROP PROCEDURE IF EXISTS load_records_to_index;";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.List;
+
+/**
+ * Remove {@code load_records_to_index} stored procedure
+ *
+ * @author nollymar
+ */
+public class Task05225RemoveLoadRecordsToIndex extends AbstractJDBCStartupTask {
+
+    private final String POSTGRES_SCRIPT = "DROP FUNCTION IF EXISTS load_records_to_index;";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import java.util.List;
+
+/**
+ * Remove {@code load_records_to_index} stored procedure
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: I think we should add here detail about what this task do

for example

- remove the load_records_to_index function if it exists

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
+package com.dotmarketing.startup.runonce;
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: is it possible do a test?, maybe we can run the Task manually and check that the data base have the change that we expect

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+                + "CLOSE cur1;\n"
+                + "\n"
+                + "SELECT * FROM tmp_records_reindex;\n"
+                + "END;";
+    }
+
+    private boolean procedureExists() throws DotDataException {
+        String query = null;
+
+        switch (DbConnectionFactory.getDBType()){
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+        dotConnect.executeStatement(script);
+
+    }
+
+    private String getOracleScript() {
+        return "CREATE OR REPLACE FUNCTION load_records_to_index(server_id VARCHAR2, records_to_fetch NUMBER, priority_level NUMBER)\n"
+                + "   RETURN types.ref_cursor IS\n"
+                + " cursor_ret types.ref_cursor;\n"
+                + " data_ret reindex_record_list;\n"
+                + "BEGIN\n"
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [The String literal "BEGIN\n" appears 4 times in this file; the first occurrence is on line 62](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05205UpdateIndexNameLengthTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    private final String POSTGRES_SCRIPT = "DROP FUNCTION IF EXISTS load_records_to_index;";
+
+    private final String MYSQL_SCRIPT = "DROP PROCEDURE IF EXISTS load_records_to_index;";
+
+    private final String MSSQL_SCRIPT =
+            "IF EXISTS (SELECT * FROM sys.objects WHERE type = 'P' AND name = 'load_records_to_index')\n"
+                    + "DROP PROCEDURE load_records_to_index;";
+
+
+    private final String ORACLE_SCRIPT = "DROP FUNCTION load_records_to_index";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05225RemoveLoadRecordsToIndexTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+        dotConnect.setSQL(query);
+        return (Map<String, String>)dotConnect.loadResults().get(0);
+
+    }
+
+    @Test
+    public void testUpgradeTaskShouldPass() throws DotDataException {
+        final Task05205UpdateIndexNameLength task = new Task05205UpdateIndexNameLength();
+
+        task.executeUpgrade();
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Looks like this run on a already altered db. We could put it in the previous state and run the upgrade task to actually test it did the modification.

Comment (nollymar) [reply dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Done

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private Map<String, String> getColumnProperties() throws DotDataException {
+        String query;
+
+        if (!DbConnectionFactory.getDBType().equals("Oracle")){
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [The String literal "Oracle" appears 4 times in this file; the first occurrence is on line 25](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+        final DotConnect dotConnect = new DotConnect();
+        dotConnect.setSQL(query);
+        return (Map<String, String>)dotConnect.loadResults().get(0);
+
+    }
+
+    private void setStateBeforeUpgrade() throws SQLException {
+        String query = null;
+
+        switch(DbConnectionFactory.getDBType()){
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [The String literal "BEGIN\n" appears 4 times in this file; the first occurrence is on line 62](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05205UpdateIndexNameLengthTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+            case "Oracle":
+                query = "select * from dba_objects where object_type = 'FUNCTION' and  object_name='LOAD_RECORDS_TO_INDEX'";
+                break;
+
+            case "Microsoft SQL Server":
+                query = "select * from sys.objects obj where obj.type in ('P', 'X') and obj.name = 'load_records_to_index'";
+                break;
+        }
+
+        final DotConnect dc = new DotConnect();
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Avoid variables with short names like dc](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 10 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 13 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 15 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Avoid variables with short names like dc](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 10 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 14 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 12 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [The String literal "Oracle" appears 4 times in this file; the first occurrence is on line 25](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [The String literal "BEGIN\n" appears 4 times in this file; the first occurrence is on line 62](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 15 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Avoid variables with short names like dc](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [The String literal "BEGIN\n" appears 4 times in this file; the first occurrence is on line 62](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 14 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 12 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [The String literal "Oracle" appears 4 times in this file; the first occurrence is on line 25](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java (hunk 13 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java]: Codacy Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java (hunk 10 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndexTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)

**Review by jdotcms — APPROVED** [review]

**Pull request merged**